net-misc/gnome-remote-desktop/gnome-remote-desktop-45_rc.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson systemd xdg
+
+DESCRIPTION="Remote desktop daemon for GNOME using pipewire"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-remote-desktop"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="aac doc +rdp systemd +vnc"
+REQUIRED_USE="|| ( rdp vnc ) aac? ( rdp )"
+RESTRICT="test" # Tests run xvfb-run directly
+
+DEPEND="
+	x11-libs/cairo
+	x11-libs/libdrm
+	>=dev-libs/libei-1.0.901
```

Comment:
We've talked about dependency ordering, right? It should try to match the order in the upstream build system, so that it's easy to compare with the dependencies in the ebuild.

```diff
@@ -21,6 +23,7 @@ epoxy_dep = dependency('epoxy')
 glib_dep = dependency('glib-2.0', version: '>= 2.68')
 gio_dep = dependency('gio-2.0')
 gio_unix_dep = dependency('gio-unix-2.0')
+libei_dep = dependency('libei-1.0', version: libei_req)
 libnotify_dep = dependency('libnotify')
 libsecret_dep = dependency('libsecret-1')
 pipewire_dep = dependency('libpipewire-0.3', version: '>= 0.3.49')
```